x/posts/simulation/operations.go

```diff
@@ -22,7 +22,7 @@ const (
 	OpWeightMsgAnswerPoll       = "op_weight_msg_answer_poll"
 	OpWeightMsgRegisterReaction = "op_weight_msg_register_reaction"
 
-	DefaultGasValue = 800000
+	DefaultGasValue = 1800000
```

Comment:
Why was this incremented?

Reply:
new checks require more gas in some cases

Reply:
Can we try making this 1.000.000 instead of 1.800.000? This would result in an increment of `200.000` instead of `1.000.000` which I hope it's sufficient